docs/content/config/advanced/tips/server-multiple-ip-addresses.md

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Use Cases | Binding DMS to a specific network'
+hide:
+  - toc
+---
+
+!!! warning "Advice not extensively tested"
+
+    This configuration has only been used with `network: host`, where you have direct access to the host interfaces.
+
+If your host system is running multiple IPv4 and IPv6 IP-addresses, you probably have an interest to
+bind outgoing SMTP connections to specific IP-addresses to ensure MX records are aligned with
+PTR-records in DNS when sending emails to avoid getting blocked by SPF for example.
```

Comment:
```suggestion
If your Docker host is running multiple IPv4 and IPv6 IP-addresses, it may be beneficial to bind outgoing SMTP connections to specific IP-address / interface. When a mail is sent outbound from DMS, it greets the MTA it is connecting to with a EHLO (DMS FQDN) which might be verified against the IP resolved, and that a `PTR` record for that IP resolves an address back to the same IP. If DMS connections are inconsistent with the IP used here, these DNS checks are likely to fail.
```

[`reject_unknown_sender`](https://github.com/docker-mailserver/docker-mailserver/issues/3716#issuecomment-1868033234) is a setting DMS has which other mail servers may also check for, where a foreign sender address is checked for having a valid DNS MX or A record. The linked issue is an SPF failure related to that.

There are two related restrictions Postfix can configure related to the suggestion that other MTAs may also check for.

> [**`reject_unknown_reverse_client_hostname`**](https://www.postfix.org/postconf.5.html#reject_unknown_reverse_client_hostname):
> **Reject** the request when the **client IP address** has **no address -> name mapping**.
> This is a weaker restriction than the [`reject_unknown_client_hostname`](https://www.postfix.org/postconf.5.html#reject_unknown_client_hostname) feature, which requires not only that the **address -> name** and **name -> address** mappings exist, but _also_ that **the two mappings reproduce the client IP address**.

---

SPF only comes into play when it authorizes a mail server via `mx` / `ip4` / `ip6` / `a` etc, as [demonstrated in my prior review comment](https://github.com/docker-mailserver/docker-mailserver/pull/3465/files#r1294061262). Likewise those DNS records related changes would need to have consistent IP.